src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -107,6 +106,35 @@ Self-service VPC peering setup is not supported for CockroachDB {{ site.data.pro
 
 To establish an AWS PrivateLink connection, refer to [Managing AWS PrivateLink for a cluster]({% link cockroachcloud/aws-privatelink.md %}). After the connection is established, you can use it to [connect to your cluster](#connect-to-your-cluster).
 
+#### Azure Private Link
+
+{{site.data.alerts.callout_success}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+1. Navigate to your cluster's **Networking > Private endpoint** tab.
+1. Click **Add a private endpoint**. Copy the value provided for **Alias**. Do not close this browser window.
+1. In a new browser window, log in to Azure Console and create a new private endpoint for your cluster.
```

Comment:
nit: log in to _the_ Azure Console?

Reply:
No, it mirrors "Log in to Google Cloud Console" I think.